index.html

```diff
@@ -12,12 +12,12 @@
     <title>Portfolio</title>
 </head>
```

Comment:
- [ ] Instead of using Portfolio please make your title something like **Benjamin's Portfolio** because it is your portfolio and you should proudly say that :blush: